package.json

```
@@ -12,11 +12,10 @@
   },
   "devDependencies": {
     "browser-sync": "2.27.10",
-    "eslint": "8.15.0",
+    "eslint": "^8.15.0",
```

Версии зависимостей оставляем зафиксированными, просто удали "крышечку"
```suggestion
    "eslint": "8.15.0",
```